File: app/src/main/java/za/co/jpsoft/winkerkreader/data/ListViewSelected.java
package za.co.jpsoft.winkerkreader.data;

/**
 * Created by Pieter Grobler on 01/09/2017.
 */

class ListViewSelected {
    private int _id;
    private Boolean smsto;

    public ListViewSelected(int _id, Boolean smsto) {
        this._id = _id;
        this.smsto = smsto;
    }

    public void set_id (int _id) {
        this._id = _id;
    }

    public void set_smsto (Boolean smsto) {
        this.smsto = smsto;
    }

    public Boolean get_smsto() {
        return this.smsto;
    }
}

File: app/src/main/kotlin/ListViewSelected.kt
package za.co.jpsoft.winkerkreader.data

/**
 * Created by Pieter Grobler on 01/09/2017.
 */
internal class ListViewSelected(id: Int, smsto: Boolean) {
    var id: Int = id
        set(value) {
            field = value
            // Note: setter name will be setId() in Java
        }

    var smsto: Boolean = smsto
    // Java getter: getSmsto(), setter: setSmsto()
}